libs/dkg/src/rbc.cpp

```diff
@@ -145,7 +145,7 @@ void RBC::Broadcast(const RBCEnvelop &env)
 }
 
 /**
- * Places a serialised message to be broadcasted via RBC into a RBCEnvelop and broadcats
+ * Places a serialised message to be broadcasted via RBC into a RBCEnvelop and broadcasts
```

Comment:
`RBCEnvelope`? `envelope` in the line below?

Reply:
I believe that envelop should become envelope but it's not a priority